src/platform/qt/library/LibraryGrid.cpp

```diff
@@ -0,0 +1,86 @@
+/* Copyright (c) 2014-2017 waddlesplash
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+#include "LibraryGrid.h"
+
+#include "ui_LibraryGrid.h"
+
+namespace QGBA {
+
+// Game banner image size
+static const quint32 GRID_BANNER_WIDTH = 144;
```

Comment:
No, and I should probably remove them anyway because they're totally wrong (I think these are the numbers from Dolphin.)